README.md

```diff
@@ -31,3 +31,40 @@ await entityClient.createEntity('contact', { first_name: 'Example', last_name: '
 
 - [epilot-sdk Docs](https://docs.epilot.io/docs/architecture/sdk)
 - [epilot-sdk Reference](https://docs.epilot.io/api)
+
+# Contributing
+
+## Updating a client against a new API spec
```

Comment:
Since these instructions are in the public repo, would it make sense to mention that these steps are applicable only to ePilot employees?

Reply:
Valid, added that note 👌